harness/determined/common/experimental/determined.py

```diff
@@ -306,8 +327,14 @@ def get_models(
                 include models with names matching this parameter.
             description: If this parameter is set, models will be filtered to
                 only include models with descriptions matching this parameter.
-            model_id: If this paramter is set, models will be filtered to
+            model_id: If this parameter is set, models will be filtered to
                 only include the model with this unique numeric id.
+            workspace_names: Workspace names to filter models by.
+            workspace_ids: Workspace IDs to filter models by.
+
+        Note:
+            This method returns an Iterable type that lazily instantiates response objects. To
```

Comment:
The word "fetch" here might mean something different to me the reader than you'd meant it as the writer. My reading of the implementation is that all models are fetched at once here, too, but only instantiated lazily as you say.